docs/components/google/google-account-picker.md

```diff
@@ -0,0 +1,38 @@
+# Google Account Picker
+
+## Overview
+A Google Account Picker is a non-visible component which can be used to pick a Google (Gmail) account registered in the user's device. This component can be used to create a login/register like system in the app.
+<br><br>
+![Google Account Picker](/assets/images/components/google/google-account-picker/preview.gif){: class="preview-img" }

```

Comment:
The images you've linked don't exist in the repo so you'll need to upload them.

Reply:
I'm new to this. Can you please tell me how to upload images.

Reply:
I'm new to this. Can you please tell me how to upload images.

Reply:
This should help you:
https://help.github.com/articles/adding-a-file-to-a-repository/

Reply:
I'm unable to upload files. It is showing this error:
![screenshot_2018-11-15-21-43-10-092_com android chrome](https://user-images.githubusercontent.com/40118018/48565909-9d78d580-e91f-11e8-8bea-aee95304c974.png)

Reply:
You need to upload the images in the branch you want to push. In this case Shreyash193:patch-1

https://github.com/Shreyash193/Documentation/tree/patch-1/docs/components/google

Reply:
Done. ✓